### ISSUE TITLE
acme.sh alias not working under FreeBSD

### ISSUE THREAD

**Issue by H4ndl3** [Neilpang/acme.sh#169]
I installed acme.sh by running `curl https://get.acme.sh | sh` but the alias wasn't working afterwards. I logged out and back in and even restarted the machine just to be sure but it still didn't work. Maybe it is because the alias command under FreeBSD needs to be `alias acme.sh '~/.acme.sh/acme.sh'` instead of `alias acme.sh=~/.acme.sh/acme.sh`? After I ran the former command it was working correctly.

**Comment by Neilpang** [comment]
If your `~/.profile` was not loaded automatically by `sh`.  you should load it first before using alias.
```
.   ~/.profile

acme.sh  --issue ..............
```

**Comment by Neilpang** [comment]
It seems that  `sh`  on freebsd doesn't load  `.profile`.  Here is the solution:
Edit your `~/.cshrc` file.  Add the following  line to the end:
```
setenv  ENV  $HOME/.profile
```
Then , log out, and log in again.

**Comment by H4ndl3** [comment]
That doesn't work either. Maybe because `acme.sh.env` contains
```
LE_WORKING_DIR="/root/.acme.sh"
alias acme.sh="/root/.acme.sh/acme.sh"
```
which doesn't work under FreeBSD. The first command doesn't work at all (maybe it should be `setenv LE_WORKING_DIR="root/.acme.sh"`?) and the second doesn't because of what I described in my first post. Is there a possible way to find out if the user is running FreeBSD and change the install behavior accordingly?

**Comment by Neilpang** [comment]
you should use `sh`  first.

```
sh
acme.sh

```

**Comment by H4ndl3** [comment]
Ah, that does the trick, thank you! How can I get it working while using `csh` as shell?

**Comment by Neilpang** [comment]
add the following line to your  `.cshrc` file:
```
alias acme.sh   /root/.acme.sh/acme.sh
setenv LE_WORKING_DIR  "/root/.acme.sh"

```

**Comment by Neilpang** [comment]
fixed 94dc5f330c735b0311353d4a627760fb3814f395

FreeBSD users are happy now.

Alias "acme.sh" is out-of-box for use for FreeBsd now.
